src/SprykerEco/Zed/CrefoPayApi/Business/Builder/Request/CancelRequestBuilder.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\CrefoPayApi\Business\Builder\Request;
+
+use Generated\Shared\Transfer\CrefoPayApiRequestTransfer;
+
+class CancelRequestBuilder extends AbstractRequestBuilder implements CrefoPayApiRequestBuilderInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\CrefoPayApiRequestTransfer $requestTransfer
+     *
+     * @return array
+     */
+    protected function convertRequestTransferToArray(CrefoPayApiRequestTransfer $requestTransfer): array
+    {
+        return $requestTransfer->getCancelRequest()->toArray(true, true);
```

Comment:
Is it okey for payment provider to receive not defined parameters?
Transfer objects are extendable, so it's possible to get more.
Also, is it safe to rely on `CamelCase` conversion from `AbstractTransfer`
My idea - to have implicit conversion here like `'CustomerName' => $tt->getCustromerName()`